bundles/org.openhab.binding.mihome/README.md

```diff
@@ -110,13 +110,17 @@ Just follow the instructions in ["Connecting devices to the gateway"](#connectin
 
 - The binding requires port `9898` to not be used by any other service on the system.
 - Make sure multicast traffic is correctly routed between the gateway and your openHAB instance
+- To correctly receive multicast traffic, when your OpenHAB machine is using multiple network interfaces, you might need to configure the optional `interface` property on the `Bridge` Thing, like so:
```

Comment:
Please check all.
```suggestion
- To correctly receive multicast traffic, when your openHAB machine is using multiple network interfaces, you might need to configure the optional `interface` property on the `Bridge` Thing, like so:
```

Reply:
Reviewed the whole file and updated all relevant instances. Also fixed some other obvious typos.